### DISCUSSION TITLE
Accessing old documentation?

### DISCUSSION THREAD

**Discussion by adityakavalur** [idaholab/moose#26969] [Q&A Getting Started] [answered]
Hello,

I was trying to refresh my memory on something and referred to your examples after a while. However, it looks like your block structure has changed a bit after the latest tagged release, lets take the most basic example found [here](https://mooseframework.inl.gov/modules/solid_mechanics/tutorials/introduction/step01.html). It looks like `[Modules/TensorMechanics/Master]` has changed to `[Physics/SolidMechanics/QuasiStatic]` . However, this produces the error for me since I am not on `next` but  `2024-02-12` tagged release. If I change that directive back to the `[Modules/TensorMechanics/Master]` the example runs fine. 

Am I missing something obvious about how to access old documentation online? I am currently building test_docs to create my local copy tied to the tagged release, so I think I should be fine, but it would still be good to know.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Old documentation is not accessible online.
You have to build it locally. Go in modules/combined and build the executable there
then do in modules/doc and run `python moosebuild.py build --serve`
after the website is generated locally, it will give you a URL to visit

Guillaume

**Reply by adityakavalur** [reply]
Thanks! yes that makes sense, its useful to have `moosedocs.py`. I was curious if I was missing some ReadTheDocs style version tab in documentation.
